troughs_and_waves/gaussian/tests/test_gaussian_data_source.py

```diff
@@ -0,0 +1,57 @@
+#  (C) Copyright 2010-2020 Enthought, Inc., Austin, TX
+#  All rights reserved.
+
+import unittest
+
+from unittest import mock
+
+from traits.testing.api import UnittestTools
+from force_bdss.api import DataValue, Slot, BaseDataSourceFactory
+
+from troughs_and_waves.gaussian.gaussian_data_source import (
+    GaussianDataSource
+)
+
+from troughs_and_waves.gaussian.gaussian_data_source_model import (
+    GaussianDataSourceModel
+)
+
+
+class TestPowerEvaluatorDataSource(unittest.TestCase, UnittestTools):
+    def setUp(self):
+        super(TestPowerEvaluatorDataSource, self).setUp()
```

Comment:
I don't think we need this super class call here since `UnittestTools` doesn't implement a `setUp` method